stream-chat-android/src/main/java/com/getstream/sdk/chat/view/MessageListView.kt

```diff
@@ -389,6 +420,34 @@ public class MessageListView : ConstraintLayout {
         setMessageListItemAdapter(adapter)
     }
 
+    public fun setLoadingView(view: View, layoutParams: FrameLayout.LayoutParams = defaultChildLayoutParams()) {
+        loadingViewContainer.removeView(loadingView)
+        loadingView = view.apply { id = LOADING_VIEW_ID }
```

Comment:
What's the use of setting these IDs on the Views that are set?

Reply:
It seems they are required only on `RecyclerView`. Removing them from empty state and loading views.